src/voodoo/types.ts

```diff
@@ -24,6 +29,15 @@ export type EncryptedVoodooMessage = {
   ciphertext: string
 }
 
+// The object to be JSON-serialized to form the body(plaintext) of the invite
+export type VoodooInvite = {
+  topic: string
+  // Addresses of the two wallets in the conversation
+  // not necessarily the two voodoo instances in a single 1:1 session
+  // e.g. Me1 sending my messages in convo to Me2
+  participantAddresses: string[]
```

Comment:
Slightly confused by this comment

In the case of Me1 -> Me2 would the participant addresses would read
['0x000ME','0x000YOU'] correct? not ['0x000ME','0x000ME']

Reply:
Yup, that is correct. It would be ['0x000ME', '0x000YOU']. Let me reread the comment and reword if necessary